app/models/collection.rb

```diff
@@ -4,6 +4,7 @@ class Collection < ActiveRecord::Base
   include HasSettingsAttribute
 
   has_and_belongs_to_many :browse_entries
+  has_one :page_landing, class_name: 'Page::Landing', dependent: :destroy
```

Comment:
I'm tempted to call this association `landing_page`, alleviating the need for the method of that name below, and which is more readable.